.circleci/config.yml

```diff
@@ -0,0 +1,48 @@
+# 1. Set your GitHub default branch to dev/develop
+# 2. In circleci project settings->advanced set:
+#    - GitHub Status Updates (this should be on already)
+#    - Only build pull requests
+#    - Auto-cancel redundant builds
+#    - Free and Open Source (this should be on already)
+# 3. If existing project with quay.io, remove the quay build triggers.
+#    - as circleci is pusing the image
+#    - reevaluate when/if quay.io adds ability to authenticate pulls
+
+version: 2.1
+jobs:
+  build:
+    environment:
+      IMAGE_NAME: quay.io/wtsicgp/gripss
+    docker:
+      - image: circleci/buildpack-deps:stretch
+    steps: 
+      - checkout
+      - setup_remote_docker
+      - run:
+          name: Build Docker image
+          command: |
+            set -e
+            BRANCH_OR_TAG="${CIRCLE_TAG:-$CIRCLE_BRANCH}"
```

Comment:
The quay.io push fails as you need to do the docker logins like cgpPindel does:

https://github.com/cancerit/cgpPindel/blob/40b1e8a7e4a7774660c32513673fecd538d51a0b/.circleci/config.yml#L22-L23